lib/shore/customers/customer_endpoints.rb

```diff
@@ -17,11 +17,11 @@ def get_customers(oid, options = {})
         params = options.select do |key, _value|
           %w(page).include?(key)
         end
-        format_response(conn.get(path, params))
```

Comment:
if you choose to remove the `format_response` method, the response will be an object - instead of the hash - with a parsed body, and this object will have the attributes of `status`, `headers` and `body`